src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## March 14, 2023
+
+### Android Mobile Core 2.1.0
+
+* Added a new API to the LaunchRulesEngine class that enables adding new rules without replacing the existing ones. 
+
+## March 13, 2023
+
+### iOS AEPCore 3.8.0
```

Comment:
nit: `### iOS Mobile Core 3.8.0`

Reply:
We have been using AEPCore for iOS until now.

Reply:
I suppose we should fix it then. We are not using AEP prefixes with any of the other extensions.

Reply:
Yeah, consistency for names is IMO better. If all the other ones don't use AEP, this should follow and be iOS Mobile Core instead.